util/perl/OpenSSL/Test.pm

```diff
@@ -1065,13 +1101,6 @@ sub __data_dir {
     return catdir($directories{SRCDATA},@_);
 }
 
-sub __results_file {
-    BAIL_OUT("Must run setup() first") if (! $test_name);
```

Comment:
You dropped this line.   Don't.

Reply:
I did so deliberately because this check is anyway done by `result_dir()` called by this function.
Isn't that sufficient?

Reply:
All right, anyway re-add that BAIL_OUT.